docs/RunningHPCCinaVirtualMachine/RunningHPCCinaVirtualMachine.xml

```diff
@@ -231,65 +231,6 @@
           <listitem>
             <para>Start the VM VirtualBox application.</para>
           </listitem>
-
-          <listitem>
-            <para>From the <emphasis role="bold">File</emphasis> menu, select
-            <emphasis role="bold">Preferences</emphasis>...</para>
-
-            <para>The <emphasis>VirtualBox - Settings</emphasis> dialog
-            displays.<figure id="VBoxSettings">
-                <title xreflabel="welc">VirtualBox - Settings</title>
-
-                <mediaobject>
-                  <imageobject>
-                    <imagedata fileref="images/vmimg14.jpg" />
-                  </imageobject>
-                </mediaobject>
-              </figure></para>
-          </listitem>
-
-          <listitem>
-            <para>Select<emphasis role="bold"> Network</emphasis> from the
-            list on the left side of the window.</para>
-          </listitem>
-
-          <listitem>
-            <para>Double-click on the <emphasis>VirtualBox Host-Only Ethernet
-            Adaptor</emphasis>.</para>
-          </listitem>
-
-          <listitem>
-            <para>From the<emphasis> Network Details</emphasis> window, select
-            the <emphasis role="bold">DHCP Server</emphasis> Tab and check the
-            <emphasis role="bold">Enable Server</emphasis> box. <figure
-                id="NetworkDetails">
-                <title xreflabel="welc">Network Details</title>
-
-                <mediaobject>
-                  <imageobject>
-                    <imagedata fileref="images/vmimg15.jpg" />
```

Comment:
If image is no longer used, the jpg needs to be deleted

Reply:
good point. Deleted images.